_posts/2019-01-20-why-we-run-our-blog.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: epic
+title: "Why We Run Our Own Blog"
+date: "2019-01-20"
+author: [orta]
+categories: [tooling, dependencies]
+comment_id: 524
+---
+
+This blog just passed the 7 year mark from our initial ["Hello World"][hw] post. We've always built and hosted our
+own blog, initially [using OctoPress][octo] but eventually migrating to just plain old Jekyll. This means our 221
+posts are just plain markdown files in a GitHub repo.
+
+Artsy uses 3 separate editorial platforms now, we built our own for Artsy Magazine, use Medium for our [Life at
+Artsy blog][laab] and Jekyll for the engineering blog. There was a healthy debate about whether we would migrate to
+one, or two systems, but I had pretty strong opinions on migrating the engineering blog to Medium and nipped that
+in the bud pretty quickly.
+
+With [Signal vs Noise][svn] being a high profile of a example of migrating to Medium and back again, I thought it's
+worth taking the time to examine our reasoning for doing it ourselves.
+
+<!-- more -->
+
+## Dependencies
+
+Almost all parts in the process of creation rely on you depending on others. For this blog, that we rely on
+GitHub's static site hosting, RubyGems + Bundler and Jekyll. Luckily for us, there are powerful incentives for
+those projects to continue long into the future.
```

Comment:
It might even be worth explicitly calling out here that those dependencies are optional:

- If GitHub pages disappears, you don't care, just switch Netfly or plain S3 with CI
- If RubyGems disappears, there will be (or is) an alternative
- If Jekyll stops being maintained, you'll still be able to use it

Top of all: you have the raw markdown files, so you'll be fine no matter what happens